Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN \
   apt-get -y --force-yes --no-install-recommends install libfontconfig curl ca-certificates git jq && \
   curl https://grafanarel.s3.amazonaws.com/builds/grafana_${GRAFANA_VERSION}_amd64.deb > /tmp/grafana.deb && \
   dpkg -i /tmp/grafana.deb && \
-  rm -f /tmp/grafana.deb && \
-  curl -L https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64 > /usr/sbin/gosu && \
-  chmod +x /usr/sbin/gosu && \
-  for plugin in $(curl -s https://grafana.net/api/plugins?orderBy=name | jq '.items[] | select(.internal=='false') | .slug' | tr -d '"'); do grafana-cli --pluginsDir "${GF_PLUGIN_DIR}" plugins install $plugin; done && \
+  rm -f /tmp/grafana.deb
+
+RUN  curl -L https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64 > /usr/sbin/gosu && \
+  chmod +x /usr/sbin/gosu
+
+RUN \
+  for plugin in $(curl -s https://grafana.net/api/plugins?orderBy=name | jq '.items[] | select(.internal=='false') | .slug' | tr -d '"'); \
+    do rc=1;count=0; while [ $rc -ne 0 ] && [ $count -lt 5 ]; do grafana-cli --pluginsDir "${GF_PLUGIN_DIR}" plugins install $plugin;rc=$?;count=$(( count+1 )); done; done
```

Comment:
Some spaces after `;` are missing. How it fail if plugin server is not available?